[Bug] - Regular crashes s3log.*FileLogger.Log

**Describe the bug**
I have been having regular crashes of a versitygw container running in Podman, I do not know what the trigger is.

**To Reproduce**
Unknown steps to reproduce. 

**Expected behavior**
I would expect the error to be handled gracefully.

**Server Version**
This has affected the official docker containers `versity/versitygw:v1.0.9` (sha256:39b58341f7666f9cc255415a2b4c993706c4892423033e40a2f9ec916366f8d1) and `versity/versitygw:v1.0.13` (sha256:dd880fd85c3faba0411da8798d3f6f0827a779d962f257f3624b1fef76d56b7b)

From inside the container:
```
/app # ./versitygw -version
Version  : v1.0.13
Build    : 63fd82654d9fa618fd11e06011495174534af5cf
BuildTime: 2025-04-25T20:44:38Z
/app # uname -a
Linux 8ff7d873558b 5.14.0-503.38.1.el9_5.x86_64 #1 SMP PREEMPT_DYNAMIC Sun Apr 13 22:01:49 EDT 2025 x86_64 Linux
```

**Additional context**
After looking at the access logs, I believe it is caused by someone at my organization running a scheduled network security scan. I'm waiting to hear back and see if they know what the scanner might be doing to crash the application. It appears completely unrelated to S3 functionality because my S3 backups are not running at the time of the crash.

**Logs**
```
No IAM service configured, enabling single account mode
initializing S3 access logs with '/log/access.log' file
caught signal hangup
caught signal hangup
caught signal hangup
caught signal hangup
caught signal hangup
panic: runtime error: slice bounds out of range [2:1]

goroutine 1403 [running]:
github.com/versity/versitygw/s3log.(*FileLogger).Log(0xc0001bc3c0, 0xc0001c4608, {0x1813820, 0xc000298c30}, {0x0?, 0x0, 0xc000647090?}
, {{0x0, 0x0}, 0x0, ...})
        /app/s3log/file.go:71 +0x9bb
github.com/versity/versitygw/s3api/controllers.SendResponse(0xc0001c4608, {0x1813820, 0xc000298c30}, 0xc000647240)
        /app/s3api/controllers/base.go:3835 +0xe6
github.com/versity/versitygw/s3api/middlewares.sendResponse(0x14e14a0?, {0x1813820?, 0xc000298c30?}, {0x1823300?, 0xc0001bc3c0?}, 0xc0
00647348?)
        /app/s3api/middlewares/authentication.go:181 +0x70
github.com/versity/versitygw/s3api.New.VerifyV4Signature.func5(0xc0001c4608)
        /app/s3api/middlewares/authentication.go:58 +0x1069
github.com/gofiber/fiber/v2.(*Ctx).Next(0xc0001c4608?)
        /go/pkg/mod/github.com/gofiber/fiber/v2@v2.52.6/ctx.go:1025 +0x3d
github.com/versity/versitygw/s3api.New.VerifyPresignedV4Signature.func4(0xc0001c4608)
        /app/s3api/middlewares/presign-auth.go:34 +0x72f
github.com/gofiber/fiber/v2.(*Ctx).Next(0xc0001c4608?)
        /go/pkg/mod/github.com/gofiber/fiber/v2@v2.52.6/ctx.go:1025 +0x3d
github.com/versity/versitygw/s3api.New.DecodeURL.func2(0xc0001c4608)
        /app/s3api/middlewares/url-decoder.go:34 +0x173
github.com/gofiber/fiber/v2.(*App).next(0xc0003dcf08, 0xc0001c4608)
        /go/pkg/mod/github.com/gofiber/fiber/v2@v2.52.6/router.go:145 +0x1a8
github.com/gofiber/fiber/v2.(*App).handler(0xc0003dcf08, 0x4d25b3?)
        /go/pkg/mod/github.com/gofiber/fiber/v2@v2.52.6/router.go:172 +0x69
github.com/valyala/fasthttp.(*Server).serveConn(0xc00015ab48, {0x182b990, 0xc000f84198})
        /go/pkg/mod/github.com/valyala/fasthttp@v1.60.0/server.go:2394 +0x11cf
github.com/valyala/fasthttp.(*workerPool).workerFunc(0xc000358510, 0xc0024f2000)
        /go/pkg/mod/github.com/valyala/fasthttp@v1.60.0/workerpool.go:225 +0x92
github.com/valyala/fasthttp.(*workerPool).getCh.func1()
        /go/pkg/mod/github.com/valyala/fasthttp@v1.60.0/workerpool.go:197 +0x32
created by github.com/valyala/fasthttp.(*workerPool).getCh in goroutine 9
        /go/pkg/mod/github.com/valyala/fasthttp@v1.60.0/workerpool.go:196 +0x194
```

--- a/s3event/event.go
+++ b/s3event/event.go
@@ -141,7 +141,12 @@ func InitEventSender(cfg *EventConfig) (S3EventSender, error) {
 
 func createEventSchema(ctx *fiber.Ctx, meta EventMeta, configId ConfigurationId) EventSchema {
 	path := strings.Split(ctx.Path(), "/")
-	bucket, object := path[1], strings.Join(path[2:], "/")
+
+	var bucket, object string
+	if len(path) > 1 {
+		bucket, object = path[1], strings.Join(path[2:], "/")
+	}
+
 	acc := ctx.Locals("account").(auth.Account)
 
 	return EventSchema{

--- a/s3log/file.go
+++ b/s3log/file.go
@@ -68,7 +68,10 @@ func (f *FileLogger) Log(ctx *fiber.Ctx, err error, body []byte, meta LogMeta) {
 	access := "-"
 	reqURI := ctx.OriginalURL()
 	path := strings.Split(ctx.Path(), "/")
-	bucket, object := path[1], strings.Join(path[2:], "/")
+	var bucket, object string
+	if len(path) > 1 {
+		bucket, object = path[1], strings.Join(path[2:], "/")
+	}
 	errorCode := ""
 	httpStatus := 200
 	startTime := ctx.Locals("startTime").(time.Time)

--- a/s3log/webhook.go
+++ b/s3log/webhook.go
@@ -65,7 +65,10 @@ func (wl *WebhookLogger) Log(ctx *fiber.Ctx, err error, body []byte, meta LogMet
 	access := "-"
 	reqURI := ctx.OriginalURL()
 	path := strings.Split(ctx.Path(), "/")
-	bucket, object := path[1], strings.Join(path[2:], "/")
+	var bucket, object string
+	if len(path) > 1 {
+		bucket, object = path[1], strings.Join(path[2:], "/")
+	}
 	errorCode := ""
 	httpStatus := 200
 	startTime := ctx.Locals("startTime").(time.Time)

